hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/CassActionsServiceImpl.java

```diff
@@ -102,6 +107,10 @@ public void setExecutor(ManagedExecutorService executor) {
         this.executor = executor;
     }
 
+    public void setSession(Session session) {
```

Comment:
Yes, Cass* classes have some setters methods to run performance tests outside of the container. Just from a standalone class.